src/js/request.js

```diff
@@ -0,0 +1,71 @@
+import { HOST } from './constants/constants.js';
+
+export const signupRequest = async (data) => {
+  const url = `${HOST}/members`;
+
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+
+  if (!response.ok) {
+    throw new Error(response.status);
+  }
+
+  return response;
+};
+
+export const loginRequest = async (data) => {
+  const url = `${HOST}/login/token`;
+
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+
+  if (!response.ok) {
+    throw new Error(response.status);
+  }
+
+  return response.json();
+};
+
+export const userInfoRequest = async (token) => {
+  const url = `${HOST}/members/me`;
+
+  const response = await fetch(url, {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${token}`,
+      'Content-Type': 'application/json',
+    },
+  });
+
+  if (!response.ok) {
+    throw new Error(response.status);
+  }
+
+  return response.json();
+};
+
+export const checkEmailDuplicatedRequest = async (email) => {
+  const url = `${HOST}/members/check-validation?`;
+  const searchParams = new URLSearchParams();
+
+  searchParams.set('email', email);
```

Comment:
```suggestion
  const searchParams = new URLSearchParams({ email });
```

선호의 차이일수도 있겠지만, 이렇게 작성하는것이 더 실수할 여지가 적을 것 같은데 어떠신가요?

Reply:
파라미터가 하나인 경우에는 이렇게 작성 할 수도 있겠네요! 좋은 팁 감사합니다👍👍 리팩토링을 하는 과정에서 쿼리가 하나뿐이라 템플릿 리터럴로 변경하였는데, URLSearchParams 를 이용하는 것이 더 적절할까요?

Reply:
하나일땐 템플릿으로 작성해도 될것같습니다. 👍